### PR TITLE
update llama artifact

### PR DESCRIPTION
**Pull request by dan-garvey**

### CODE REVIEW

File: experimental/regression_suite/tests/pregenerated/test_llama2.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 ]
 
 llama2_7b_f16qi4_source = fetch_source_fixture(
-    "https://storage.googleapis.com/shark_tank/stella/llama2_7b_int4_stripped.mlir",
```
Comment (powderluv) [inline experimental/regression_suite/tests/pregenerated/test_llama2.py:22]: Do you want to keep a known URL in `https://storage.googleapis.com/shark_tank/iree-tests/<SHA>` or such ?

**Comment by powderluv** [comment]
I don't think you want the submodule updates

File: experimental/regression_suite/tests/pregenerated/test_llama2.py (hunk 1 above)
Comment (stellaraccident) [reply experimental/regression_suite/tests/pregenerated/test_llama2.py:22]: We ultimately want all of these coming from common places like that which we update with a generation script. However, as long as we're in this phase with this one of it being a bit hand-massaged, I'm ok with Dan taking responsibility to keep it updated.

**Review by stellaraccident — CHANGES_REQUESTED** [review]
Please revert the submodule changes (can't even see good test runs like it is).

**Comment by dan-garvey** [comment]
I don't even know how I managed to pull ancient submodules. Something about my fork being several thousand commits behind or something.

**Comment by stellaraccident** [comment]
Thanks. Looks like it got faster but a millisecond or two.

**Pull request merged**